website/source/quickstart.rst

```diff
@@ -60,6 +60,10 @@ Test the compatibility of a schema with the latest schema under subject "test-ke
     $KARAPACE_REGISTRY_URI/compatibility/subjects/test-key/versions/latest
   {"is_compatible":true}
 
+NOTE: if the subject's compatibility mode is transitive (BACKWARD_TRANSITIVE, FORWARD_TRANSITIVE or FULL_TRANSITIVE) then the
```

Comment:
Is there any other documentation that needs to be updated?

Reply:
I don't think so. The documentation should be revised completely IMO but for that PR I think its fine as it is. PS good addition